cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -592,13 +593,35 @@ define(["jquery", "common/js/spec_helpers/ajax_helpers", "common/js/components/u
                 };
 
                 setModalTimedExaminationPreferenceValues = function(
-                    is_timed_examination,
+                    is_time_limited,
+                    id_timed_exam,
                     time_limit,
-                    is_exam_proctoring_enabled
+                    is_practice_exam,
+                    is_proctored_enabled,
+                    enable_proctored_exams
                 ){
-                    $("#id_time_limit").val(time_limit);
-                    $("#id_exam_proctoring").prop('checked', is_exam_proctoring_enabled);
-                    $("#id_timed_examination").prop('checked', is_timed_examination);
+                    if (!is_time_limited) {
+                        this.$("#id_not_timed").prop('checked', true);
+                        return;
+                    }
+                    if (enable_proctored_exams) {
+                        $("#id_time_limit").val(time_limit);
+                        this.$('#id_time_limit_div').show();
```

Comment:
Why do you have to explicitly show the div? I would expect that you can click on the UI elements, and the handlers would modify the HTML.